querybook/webapp/components/QueryViewReview/statusUtils.ts

```diff
@@ -23,9 +23,9 @@ export const getStatusProps = (
     if (isRejected) {
         return {
             status: Status.error,
-            tooltip: 'Changes have been requested.',
+            tooltip: 'Query has been rejected.',
```

Comment:
can we make the wording consistent, either query review or review. query rejected could mean the query execution rejected to me.